pkg/ibmvpc/ibmDrawioGenerator.go

```diff
@@ -10,6 +10,7 @@ func (v *VPC) ShowOnSubnetMode() bool                  { return true }
 func (z *Zone) ShowOnSubnetMode() bool                 { return true }
 func (s *Subnet) ShowOnSubnetMode() bool               { return true }
 func (sgl *SecurityGroupLayer) ShowOnSubnetMode() bool { return false }
+func (sg *SecurityGroup) ShowOnSubnetMode() bool       { return false }
```

Comment:
Why was this added?

Reply:
every resource has a method  ShowOnSubnetMode()
do we show it on the canvas when we are of subnet mode?

Reply:
Yes, that I remembered, but why was this specific method added for `(sg *SecurityGroup)` in this specific PR?

Reply:
I had a bug, in which I created a Treenode per SecurityGroupLayer, instead creating a treeNode per SecurityGroup.
with the bug, SecurityGroup did not implemented the interface DrawioResourceIntf
now it does